chainer/functions/array/get_item.py

```diff
@@ -52,7 +52,16 @@ def forward(self, inputs):
         xp = cuda.get_array_module(*inputs)
         gx = xp.zeros(self._in_shape, self._in_dtype)
         if xp is numpy:
-            numpy.add.at(gx, self.slices, inputs[0])
+            try:
```

Comment:
How about using `numpy.lib.NumpyVersion` to test NumPy version?

https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.lib.NumpyVersion.html

(It's better to evaluate and hold the result in a module level variable.)